.circleci/config.yml

```diff
@@ -6,9 +6,13 @@ jobs:
     executor:
       name: node/default
       tag: '10.16'
-    
+
     steps:
       - checkout
+      - setup_remote_docker
```

Comment:
I might move these steps to right before the integration so we won't need to do this at all if the regular unittests fail.

Reply:
Done, I was accounting for redis to be ready for the integration tests.